server/athenian/api/openapi/openapi.yaml

```diff
@@ -838,6 +838,183 @@ paths:
 # {% endif %}
       x-openapi-router-controller: athenian.api.controllers.reposet_controller
       x-codegen-request-body-name: body
+  /team/{id}:
+    delete:
+      operationId: delete_team
+      parameters:
+      - description: Numeric identifier of the team to delete.
+        explode: false
+        in: path
+        name: id
+        required: true
+        schema:
+          type: integer
+        style: simple
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                type: object
+          description: Empty object indicates a successful removal.
+        403:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The user is forbidden to delete the specified team.
+        404:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The specified team was not found.
+      security:
+      - bearerAuth: []
+      summary: Delete a team.
+      tags:
+      - team
+# {% if False %}
+      - default
+# {% endif %}
+      x-openapi-router-controller: athenian.api.controllers.team_controller
+      x-codegen-request-body-name: body
+    get:
+      operationId: get_team
+      parameters:
+      - description: Numeric identifier of the team to list.
+        explode: false
+        in: path
+        name: id
+        required: true
+        schema:
+          type: integer
+        style: simple
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Team'
+          description: List of members in the specified team.
+        403:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The user is forbidden to list the specified team.
+        404:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The specified team was not found.
+      security:
+      - bearerAuth: []
+      summary: List a team's members. The user must be in the account that owns the
+        team.
+      tags:
+      - team
+# {% if False %}
+      - default
+# {% endif %}
+      x-openapi-router-controller: athenian.api.controllers.team_controller
+      x-codegen-request-body-name: body
+    put:
+      operationId: update_team
+      parameters:
+      - description: Numeric identifier of the team to update, empty on create.
+        explode: false
+        in: path
+        name: id
+        # Not actually required, but cannot set optional path parameter:
```

Comment:
@vmarkovtsev wdyt about this? Do you have a better approach?

Reply:
You set it as `required: true` so we will not be able to call `PUT /team` to create a new one. 

I had a similar problem with `/become` and I solved it with a URL parameter. However, we've got a CRUD here...

So maybe I added `CREATE /reposet` because of this issue.

Reply:
Got it, it makes sense. I'll then add an endpoint similarly to `/reposet`